pkg/apiserver/authorizer_test.go

```diff
@@ -30,8 +32,14 @@ import (
 
 	"k8s.io/apimachinery/pkg/runtime"
 	k8sfake "k8s.io/client-go/kubernetes/fake"
+	"kubevirt.io/containerized-data-importer/tests/reporters"
 )
 
+func TestAuthorizer(t *testing.T) {
```

Comment:
In general we try to have a single 'suite' file that has the 'Testing.T' in it. We name it: _suite_test.go with the package here being apiserver. It also allows us to set things like logger in the before suite. A decent example is in pkg/controller/controller_suite_test.go